docs/tutorials/linting_formatting.rst

```diff
@@ -18,21 +18,9 @@ Local Linting with Flake8 (VSC):
 Flake8 Settings
 ****************
 
-* Ignore singular line:
+* Ignore single line: Add :code:`# noqa` to the end of the line
```

Comment:
One can use `# noqa: E711` to only ignore one specific error code in that line.

Reply:
See https://github.com/qunicorn/qunicorn-core/pull/34/commits/65e7bfe019730bae6bd136402c4c6feebbe13fe5